doc/docs/modules/ROOT/pages/backup.adoc

```diff
@@ -50,34 +50,76 @@ In Neo4j 4.0, the system can be multidatabase; most systems have at least 2 DBs,
 
 First you want to create a kubernetes secret that contains the content of your account service key.  This key must have permissions to access the bucket and backup set that you're trying to restore. 
 
+**AWS**
+
+- You must create the credential file and this file should look like this:
+```aws-credentials
+[default]
+region=
+aws_access_key_id=
+aws_secret_access_key=
+```
+
+- You have to create a secret for this file
+```shell
+kubectl create secret generic neo4j-aws-credentials \
+    --from-file=credentials=aws-credentials
+```
+
+**GCP**
+
+- You must create the credential file and this file should look like this:
+```gcp-credentials.json
+{
+  "type": "",
+  "project_id": "",
+  "private_key_id": "",
+  "private_key": "",
+  "client_email": "",
+  "client_id": "",
+  "auth_uri": "",
+  "token_uri": "",
+  "auth_provider_x509_cert_url": "",
+  "client_x509_cert_url": ""
+}
+
+```
+
+- You have to create a secret for this file
 ```shell
-MY_SERVICE_ACCOUNT_KEY=$HOME/.google/my-service-key.json
-kubectl create secret generic neo4j-service-key \
-   --from-file=credentials.json=$MY_SERVICE_ACCOUNT_KEY
+kubectl create secret generic neo4j-gcp-credentials \
+    --from-file=credentials=gcp-credentials.json
 ```
 
-The backup container is going to take this kubernetes secret
-(named `neo4j-service-key`) and is going to mount it as a file
-inside of the backup container (`/auth/credentials.json`).  That
-file will then be used to authenticate the storage client that we
-need to upload the backupset to cloud storage when it's complete.
 
 ### Running a Backup
 
 The backup method is itself a mini-helm chart, and so to run a backup, you just
 do this as a minimal required example:
 
+**AWS**
+
+```shell
+helm install my-neo4j-backup . \
```

Comment:
the dot here (current directory) is misleading; user either needs to install it from `tools/backup` assuming you're in the source repo, or the .tgz file of a release must be used.